pages/portfolio.html

```diff
@@ -5,8 +5,50 @@
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
   <title>Document</title>
+  <link href="../styles/styles.css" rel="stylesheet">
 </head>
 <body>
-  
+  <header>
+    <h1>Justina Liu</h1>
+    <nav>
+      <div class="nav-flex-link" >
+          <a href="index.html">Home</a>
+      </div>
+      <div class="nav-flex-link" >
+        <a href="portfolio.html">Portfolio</a>
+      </div>
+      <div class="nav-flex-link" >
+        <a href="about.html">About</a>
+      </div>
+    </nav>
+</header>
+  <main>
+    <div class="grid-container">
```

Comment:
Consider a class name like `projects` or `project-container` to be alittle more descriptive.